src/firebolt/db/connection.py

```diff
@@ -116,86 +108,37 @@ def connect_v2(
     client = ClientV2(
         auth=auth,
         account_name=account_name,
-        base_url=system_engine_url,
         api_endpoint=api_endpoint,
         timeout=Timeout(DEFAULT_TIMEOUT_SECONDS, read=None),
         headers={"User-Agent": user_agent_header},
     )
+
     # Don't use context manager since this will be stored
     # and used in a resulting connection
     system_engine_connection = Connection(
         system_engine_url,
-        database,
+        None,
         client,
         CursorV2,
-        None,
         api_endpoint,
         system_engine_params,
     )
```

Comment:
Good point

Reply:
However, I see that closing the system engine connection will close the client and make it unusable. Otherwise, closing doesn't do anything special. I'll leave a comment explaining why we don't close it